src/fu-util-common.c

```diff
@@ -1490,8 +1490,10 @@ fu_util_plugin_flag_to_string(FwupdPluginFlags plugin_flag)
 		return NULL;
 	if (plugin_flag == FWUPD_PLUGIN_FLAG_USER_WARNING)
 		return NULL;
-	if (plugin_flag == FWUPD_PLUGIN_FLAG_REQUIRE_HWID)
-		return NULL;
+	if (plugin_flag == FWUPD_PLUGIN_FLAG_REQUIRE_HWID) {
+		/* TRANSLATORS: Plugin is active only if hardware is found */
+		return _("Enabled if hardware is found");
```

Comment:
"hardware matches" perhaps?

Reply:
that or just make it "Enabled".  This detail might not be needed for the user.